### ISSUE TITLE
Improvements over gtarcea/AngularStomp ?

### ISSUE THREAD

**Issue by ceefour** [mrolafsson/stompie#1]
Hi, thank you @mrolafsson :+1:, stompie looks very interesting :)

Will this improve upon @gtarcea's https://github.com/gtarcea/AngularStomp ?

Or perhaps there's a plan to combine forces?

**Comment by mrolafsson** [comment]
Hi @ceefour - thanks for getting in touch! I only had a cursory glance at AngularStomp at the time and for some reason I couldn't get it to work but suspect that it was a server side issue :) To be honest this was more of an experiment for me in Angular/Websockets and publishing to bower. I'd be happy to combine with @gtarcea on this. If this does not evolve beyond AngularStomp I will just remove the project. Any tips and ideas would be welcome!

**Comment by mrolafsson** [comment]
Just looked through AngularStomp, didn't realise how similar our approach ended up being. Thanks for bringing up.

**Comment by ceefour** [comment]
You're right, https://github.com/gtarcea/AngularStomp doesn't work out of the box with current SockJS. There's also subtle bugs which I had sent pull requests.

I did a few tweaks with it to make it work with SockJS, and only SockJS for now (which is why I didn't pull request these changes, at least not before I made it flexible): https://github.com/ceefour/AngularStomp

I used my version in this [face-recognition-over-AMQP-Stomp-WebSocket demo](https://www.youtube.com/watch?v=VLQT2xsRG3o).
